### DISCUSSION TITLE
Hammerspoon

### DISCUSSION THREAD

**Discussion by jissin** [elrumo/macOS_Big_Sur_icons_replacements#1262] [Icon request] [answered]
Could anyone make a icon for [Hammerspoon](http://www.hammerspoon.org) ? thx a lot

**Comment by cake03** [comment, marked as answer]
Hammerspoon already has a Big Sur icon. Do you want an alternate one?

**Reply by jissin** [reply]
Thanks for your reminding. After I updated the software, I saw the new Big Sur icon, lol
